src/main/java/org/jabref/logic/importer/CompositeIdFetcher.java

```diff
@@ -19,6 +19,10 @@ public CompositeIdFetcher(ImportFormatPreferences importFormatPreferences) {
         this.importFormatPreferences = importFormatPreferences;
     }
 
+    public CompositeIdFetcher() {
+        importFormatPreferences = null;
```

Comment:
No, you need to pass the preferences. - You have it availble in line 101 in ImportHandler.  Store it in a class variable and use the class variable in line 367 of ImportHandler.

Reply:
Oh, sorry. I see. I have deleted the constructor. I will create a pull request again. Thank you so much.

Reply:
No additional pull request. Just push. As your teammate already did!

Reply:
Okkk!thank you